config/initializers/hcaptcha.rb

```diff
@@ -0,0 +1,7 @@
+require "hcaptcha"
+
+HCaptcha.configure do |hc|
+  hc.endpoint = "https://hcaptcha.com"
+  hc.secret = "CHANGEME"
```

Comment:
We want to pull this from `Exercism.config`. Also, we might want to set up an Exercism HCaptcha account.

Reply:
> We want to pull this from Exercism.config

Cool. I'll set that up before merging this.

> Also, we might want to set up an Exercism HCaptcha account.

We have one. The token I put in the issue for this is from that account. But they don't seem to be multi-user and I created it with my GitHub so I can't give access.